starter_code/src/movies.js

```diff
@@ -5,9 +5,39 @@
 // Get the average of all rates with 2 decimals 
 
 
+function ratesAverage(elements){
+    var sum = elements.reduce(function(accumulator,item){
+        return accumulator + parseFloat(item.rate);
+    }, 0);
+    
+   return Math.round(sum / elements.length*100)/100;
+};
 // Get the average of Drama Movies
 
-
+function dramaMoviesRate(elements){
+    var count = 0;
+    var sum = elements.reduce(function(accumulator,item){
```

Comment:
This solution is fine, but if you filter the array before (letting only films with the category Drama), you can reuse the previous function (ratesAverage).